docs/content/references/ts-sdk/typescript/index.mdx

```diff
@@ -16,6 +17,9 @@ npm i @iota/iota-sdk
 
 The following table lists the locations for IOTA networks.
 
+{ /* TODO: Shhould be moved to mainnet once it is live */ }
```

Comment:
Maybe create an issue for this TODO (if there isn't already) and put a link to it instead.

also, typo :stuck_out_tongue_winking_eye:

Reply:
```suggestion
{ /* TODO: https://github.com/iotaledger/iota/issues/4497 */ }
```